share/templates/categories.json

```diff
@@ -0,0 +1,93 @@
+{
+  "categories": [
+    {
+      "title": "General",
```

Comment:
Do we have a table for translating these names?

Reply:
We should have such a table. Translations will work in the same way as they do in MU3 but instead of passing folder names to qApp->translate, we will pass strings from this JSON